pkg/agent/monitortool/monitor_test.go

```diff
@@ -354,18 +362,34 @@ func TestUpdateMonitorPingInterval(t *testing.T) {
 	assert.EventuallyWithT(t, func(t *assert.CollectT) {
 		packets = collect(packets)
 		assert.ElementsMatch(t, []string{"10.0.2.1", "10.0.3.1"}, extractIPs(packets))
+		assert.Equal(t, 0, int(reportCount.Load()), "Expected no report yet (jitter still pending)")
+	}, 2*time.Second, 10*time.Millisecond)
+
+	// Advance 1 more second → total 61s: report should now occur
+	fakeClock.Step(1 * time.Second)
+	assert.EventuallyWithT(t, func(t *assert.CollectT) {
+		assert.Equal(t, 1, int(reportCount.Load()), "Expected report after jittered interval (total 61s)")
 	}, 2*time.Second, 10*time.Millisecond)
 
+	// Clear count for next phase
+	reportCount.Store(0)
+
 	// We increase the ping interval from 60s to 90s.
 	newNLM := nlm.DeepCopy()
 	newNLM.Spec.PingIntervalSeconds = 90
 	newNLM.Generation = 1
 	_, err := m.crdClientset.CrdV1alpha1().NodeLatencyMonitors().Update(ctx, newNLM, metav1.UpdateOptions{})
 	require.NoError(t, err)
 
-	// Again, we have to wait for the second ticker to be created before we can advance the clock.
+	// Again, we have to wait for the tickers to be created before we can advance the clock.
```

Comment:
```suggestion
	// Again, we have to wait for the 2 new tickers to be created before we can advance the clock.
```

I think that's enough, and you can remove the 6 lines below